tests/experimental/datasources/test_postgres_datasource.py

```diff
@@ -713,64 +721,36 @@ def test_validate_invalid_postgres_connection_string(
             pass
 
 
-def bad_connection_string_config() -> tuple[
-    str, str, str, str | None, TestConnectionError
-]:
+def bad_connection_string_config() -> tuple[str, str, str]:
     connection_string = "postgresql+psycopg2://postgres:@localhost/bad_database"
     table_name = "good_table"
     schema_name = "good_schema"
-    mock_sqlalchemy_engine_connect_error_raised = (
```

Comment:
After comments from @billdirks, I decided to remove any assertions against error message strings, even though this strictly tests GX messages.

Reply:
I'm fine with this but just wanted to flag that `pytest.raises` can do a partial string match that would be less brittle than an exact match.
https://docs.pytest.org/en/7.2.x/reference/reference.html#pytest.raises